src/roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -49,14 +49,18 @@ class NwbImagingExtractor(ImagingExtractor):
     mode = "file"
     installation_mesg = "To use the Nwb Extractor run:\n\n pip install pynwb\n\n"  # error message when not installed
 
-    def __init__(self, file_path: PathType, optical_series_name: Optional[str] = "TwoPhotonSeries"):
+    def __init__(
+        self,
+        file_path: PathType,
+        optical_series_name: Literal["TwoPhotonSeries", "OnePhotonSeries"] = "TwoPhotonSeries",
```

Comment:
This is just the name, no need to change
```suggestion
        optical_series_name: Optional[str] = "TwoPhotonSeries"
```